src/WingetCreateCLI/Commands/UpdateCommand.cs

```diff
@@ -95,9 +95,18 @@ public override async Task<bool> Execute()
 
             try
             {
+                GitHub client = new GitHub(this.GitHubToken, this.WingetRepoOwner, this.WingetRepo);

```

Comment:
Don't you want similar code in PromptPackageIdentifierAndCheckDuplicates?

Reply:
Let me know if you agree, but I think for PromptPackageIdentifierAndCheckDuplicates, I can just set that to null. There will never be a case when we want to be using the user's github token for the duplicate check. We would always want to check the public repo and therefore don't need a specific token.

Reply:
We could definitely get away with this, but the reason I added the token to these calls was because GitHub kept throttling me after a few runs, so I had to provide a token to keep using the tool. If we get rid of it here, then the user will just have to wait until GitHub stops throttling them.

Reply:
Ahh i forgot about throttling. You're right, I added the token back and added a token check prior to prompting for packageId